testcomponent/features/find.feature

```diff
@@ -149,51 +175,56 @@ Feature: Find records
             """
 
     Scenario: Count all records
-        When I start a find operation
+        When I filter on all records
         Then I will count 3 records
-    
+
+    Scenario: Count with Limit of 0 (limit will be ignored)
+        When I filter on all records
+        And I set the limit to 0
+        Then I will count 3 records
```

Comment:
Maybe we could add a new step explicitly stating that we are not ignoring the zero limit in this scenario?

Reply:
Done